governance/proposals/udt/020-swap-arb-bridge-to-mainnet.js

```diff
@@ -0,0 +1,319 @@
+/**
+ * This proposal swaps ARB tokens for ETH on Arbitrum and bridges both UDT and ETH
```

Comment:
Please make sure we're not getting WETH (like we did last time) and if we do please unwrap the WETH first!